docs/src/main/asciidoc/datastore.adoc

```diff
@@ -26,7 +26,7 @@ dependencies {
 }
 ----
 
-We provide a link:../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-datastore[Spring Boot Starter for Spring Data Datastore], with which you can use our recommended auto-configuration setup.
+We provide a link:https://github.com/spring-cloud/spring-cloud-gcp/tree/master/spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-datastore[Spring Boot Starter for Spring Data Datastore], with which you can use our recommended auto-configuration setup.
```

Comment:
Change to `googlecloudplatform` org link instead of `spring-cloud`?

Though it might be easier to just make a note in your spreadsheet to fix all the links in one go and leave this here if you prefer.